lib/roku/index.js

```diff
@@ -7,6 +7,24 @@ var apiUrl = {
 	production: 'https://apipub.roku.com/listen/transaction-service.svc/validate-transaction'
 };
 
+function parseResult(resultString) {
+	var result = JSON.parse(resultString);
+
+	// parse non-standard date properties (i.e. /Date(1483242628000-0800)/)
+	// in order to extract value by milliseconds
+	result.expirationDate = new Date(parseInt(result.expirationDate.substr(6), 10)).getTime();
```

Comment:
Is this always set on Roku? Do we not need a `null` fallback?

Reply:
I didn't add this part or changed how the dates are handled for Roku. I just moved it over to`parseResult` function and added the already existing`purchaseDate` and `expirationDate` to root of object.

Reply:
Ah, indeed. Alright, that's not gonna block this PR then :)

FYI to @KLVTZ, is this lack of `null` not a problem?

Reply:
@ronkorving I've never seen a Roku receipt that did not have an expiration date.

We could add `null` as a fallback in case this changes.

Reply:
Added it.